### PR TITLE
kh360-react-673-lazy-load-evaluation-templates

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
Implement lazy loading on Evaluation Templates
[#673](https://github.com/nerubia/kh360-react/issues/673)

### CODE REVIEW

File: src/features/admin/evaluation-templates/evaluation-templates-dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  onClose?: (e: null | React.MouseEvent) => void
+  title?: React.ReactNode
+  description?: React.ReactNode
+  size?: "small" | "medium" | null | undefined
+  maxWidthMin?: boolean | null | undefined
+  closeBtn?: string
+  acceptBtn?: string
+}
+
+const EmailTemplatesDialog: React.FC<EvaluationTemplateDialogProps> = ({
```
Comment (gborgonia-ir) [inline src/features/admin/evaluation-templates/evaluation-templates-dialog.tsx:17]: rename also thi

File: src/features/admin/evaluation-templates/evaluation-templates-table.tsx [hunk 2]
```diff
@@ -19,6 +18,10 @@ import {
 } from "@custom-types/evaluation-template-type"
 import { Table } from "@components/ui/table/table"
 
+const EmailTemplatesDialog = lazy(
```
Comment (gborgonia-ir) [inline src/features/admin/evaluation-templates/evaluation-templates-table.tsx:21]: rename also this

File: src/features/admin/evaluation-templates/evaluation-templates-dialog.tsx (hunk 1 above)
Comment (chadiegilnerubia) [reply src/features/admin/evaluation-templates/evaluation-templates-dialog.tsx:17]: Owws mb okey2

**Review by gborgonia-ir — APPROVED** [review]

File: src/features/admin/evaluation-templates/evaluation-templates-dialog.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  title,
+  description,
+  onClose,
+  size,
+  maxWidthMin,
+  closeBtn,
+  acceptBtn,
+}) => {
+  return (
+    <Dialog open={open} size={size} maxWidthMin={maxWidthMin}>
```
Comment (emmaacha) [inline src/features/admin/evaluation-templates/evaluation-templates-dialog.tsx:29]: Please use custom dialog component here

**Review by emmaacha — CHANGES_REQUESTED** [review]

**Pull request merged**